mirror-kickstarter.sh

```diff
@@ -18,16 +20,6 @@ fi;
 ${sudo_prefix}apt-get update
 ${sudo_prefix}apt-get install -y software-properties-common
 
-echo -e "${green_color}Import fingerprint key for remote PHP mirror...${rest_color}"
-${sudo_prefix}apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 14AA40EC0831756756D7F66C4F4EA0AAE5267A6C
-
-echo "deb http://ppa.launchpad.net/ondrej/php/ubuntu $(lsb_release -sc) main" | ${sudo_prefix}tee /etc/apt/sources.list.d/ondrej.list
-echo "deb-src http://ppa.launchpad.net/ondrej/php/ubuntu $(lsb_release -sc) main" | ${sudo_prefix}tee -a /etc/apt/sources.list.d/ondrej.list
-
-echo -e "${green_color}Import key for remote Nginx mirror...${rest_color}"
-echo "deb http://ppa.launchpad.net/ondrej/nginx/ubuntu $(lsb_release -sc) main" | ${sudo_prefix}tee -a /etc/apt/sources.list.d/ondrej.list
-echo "deb-src http://ppa.launchpad.net/ondrej/nginx/ubuntu $(lsb_release -sc) main" | ${sudo_prefix}tee -a /etc/apt/sources.list.d/ondrej.list
-
```

Comment:
Why deprecating the [ondrej/php](http://ppa.launchpad.net/ondrej/php/ubuntu/)?

This repository always update the latest PHP versions and it's not old. The repository is available [here](https://github.com/oerdnj/deb.sury.org).